src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -485,12 +486,9 @@ private async Task<bool> HashAsync()
                             _hashToFilesMap[hashData.Sha256] = paths;
                         }
 
-                        _run?.GetFileIndex(new ArtifactLocation { Uri = context.TargetUri },
-                                           dataToInsert: _dataToInsert,
-                                           hashData: hashData);
-
                         paths.Add(localPath);
                         context.Hashes = hashData;
+                        _pathToHashDataMap?.Add(localPath, hashData);
                     }
```

Comment:
Updating this will enable us to use the code from SarifLogger to create the artifacts